settings.py

```diff
@@ -504,9 +504,8 @@ def JINJA_CONFIG():
     from django.core.cache.backends.memcached import MemcachedCache
     from django.core.cache import caches
     cache = caches['memcache']
-    config = {'extensions': ['jinja2.ext.i18n', 'tower.template.i18n',
-                             'jinja2.ext.with_', 'jinja2.ext.loopcontrols',
-                             'jinja2.ext.autoescape',
+    config = {'extensions': ['jinja2.ext.i18n', 'jinja2.ext.with_',
```

Comment:
Somehow that broke everything b/c `gettext` isn't in the context? Also, the example jingo test project has both - do you need both? I tried it and it failed. I'll poke with this on Monday.